tools/PowerShell/README.md

```diff
@@ -0,0 +1,179 @@
+# Windows Package Manager PowerShell Modules
+
+## History
+This set of PowerShell modules was originally built as a Microsoft Hackathon project in 2021.
+
+We started this project as an exploration to design the right set of cmdlets with PowerShell approved nouns and verbs. As we began to explore wrapping the Windows Package Manager we identified several areas of complexity. Rather than slow down progress on building PowerShell cmdlets, we decided to forge ahead and call out the areas where the experience with pipelines is sub-optimal.
+
+For example, the Windows Package Manager (later referenced as just CLI) CLI was designed for displaying output in the standard width Windows Terminal. As such, long package names and "Id"s are truncated with a single width ellipsis character. Users wanting to use the PowerShell pipeline to pass values to another cmdlet will likely encounter undesired behavior due to this truncation. We have decided to declare the "Microsoft.WinGet" module as an alpha release until these problems have been resolved. We are planning to change the status of "PreRelease" to beta once we believe we have addressed the issues related to piping the output to other cmdlets.
```

Comment:
```suggestion
For example, the Windows Package Manager (later referenced as winget or CLI) CLI was designed for displaying output in the standard width Windows Terminal. As such, long package names and "Id"s are truncated with a single width ellipsis character. Users wanting to use the PowerShell pipeline to pass values to another cmdlet will likely encounter undesired behavior due to this truncation. We have decided to declare the "Microsoft.WinGet" module as an alpha release until these problems have been resolved. We are planning to change the status of "PreRelease" to beta once we believe we have addressed the issues related to piping the output to other cmdlets.
```

Reply:
I completely agree with the feedback here.

We're going to add the capability to output data as JSON in the COM API. Then we will switch these cmdlets to use that mechanism. These cmdlets are *very* "Hackathon". My thought is that a (mostly) "working" sample at least gets the nouns and verbs out in a manner that we can share to get feedback.

I don't know what to call something that's earlier than "alpha". There seemed to be some limitations on what we could use for pre-release values.